### ISSUE TITLE
`npm run build` is failed because of `error TS2307`

### ISSUE THREAD

**Issue by wafuwafu13** [JonnyBurger/remotion#38]
1. Fork this repository
2. Clone this repository
```
~/desktop
$ git clone git@github.com:wafuwafu13/remotion.git
```
3. Add upstream branch and merge
```
~/desktop/remotion
$ git remote add upstream git@github.com:JonnyBurger/remotion.git

~/desktop/remotion
$ git fetch upstream

~/desktop/remotion
$ git merge upstream/main
```

Ref: https://github.com/JonnyBurger/remotion/blob/main/CONTRIBUTING.md#code-setup

4. `npm i && npx lerna bootstrap` (success)

<details>

<summary>log</summary>

```
~/desktop/remotion
$ npm i && npx lerna bootstrap  (success)
npm WARN eslint-plugin-react-native-normalized@1.1.0 requires a peer of eslint@^6.1.0 but none is installed. You must install peer dependencies yourself.

audited 569 packages in 3.318s

52 packages are looking for funding
  run `npm fund` for details

found 6 vulnerabilities (3 low, 3 high)
  run `npm audit fix` to fix them, or `npm audit` for details
lerna info version 2.11.0
lerna info Bootstrapping 9 packages
lerna info lifecycle preinstall
lerna info Installing external dependencies
lerna info Symlinking packages and binaries
lerna info lifecycle postinstall
lerna info lifecycle prepublish
lerna info lifecycle prepare
lerna success Bootstrapped 9 packages
```
</details>

5. `npm run build` (failed)

<details>

<summary>log</summary>

```zsh
~/desktop/remotion
$ npm run build

> @1.0.0-alpha.4 build /Users/tagawahirotaka/Desktop/remotion
> lerna run build

lerna info version 2.11.0

> create-video@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/create-video
> tsc -d && node add-bin.js


> remotion@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/core
> tsc -d


> @remotion/eslint-plugin@1.0.0 build /Users/tagawahirotaka/Desktop/remotion/packages/eslint-plugin
> tsc -d


> docs@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/docs
> docusaurus build

[en] Creating an optimized production build...
ℹ Compiling Client
ℹ Compiling Server
✔ Client: Compiled successfully in 13.82s
✔ Server: Compiled successfully in 16.17s

Success! Generated static files in build. Use `npm run serve` to test your build locally.


> @remotion/eslint-config@1.0.0 build /Users/tagawahirotaka/Desktop/remotion/packages/eslint-config
> tsc -d


> @remotion/bundler@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/bundler
> tsc -d


> @remotion/renderer@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/renderer
> tsc -d

lerna ERR! build Errored while running script in '@remotion/cli'
lerna ERR! execute callback with error
lerna ERR! Error: Command failed: npm run build
lerna ERR! npm ERR! code ELIFECYCLE
lerna ERR! npm ERR! errno 2
lerna ERR! npm ERR! @remotion/cli@1.0.5 build: `tsc -d && node add-bin.js`
lerna ERR! npm ERR! Exit status 2
lerna ERR! npm ERR!
lerna ERR! npm ERR! Failed at the @remotion/cli@1.0.5 build script.
lerna ERR! npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
lerna ERR!
lerna ERR! npm ERR! A complete log of this run can be found in:
lerna ERR! npm ERR!     /Users/tagawahirotaka/.npm/_logs/2021-02-09T23_50_38_041Z-debug.log
lerna ERR!
lerna ERR! > @remotion/cli@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/cli
lerna ERR! > tsc -d && node add-bin.js
lerna ERR!
lerna ERR! src/render.tsx(8,8): error TS2307: Cannot find module '@remotion/renderer' or its corresponding type declarations.
lerna ERR! src/render.tsx(58,29): error TS7006: Parameter 'c' implicitly has an 'any' type.
lerna ERR! src/render.tsx(76,19): error TS7006: Parameter 'f' implicitly has an 'any' type.
lerna ERR!
lerna ERR!     at /Users/tagawahirotaka/Desktop/remotion/node_modules/execa/index.js:236:11
Error: Command failed: npm run build
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! @remotion/cli@1.0.5 build: `tsc -d && node add-bin.js`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the @remotion/cli@1.0.5 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tagawahirotaka/.npm/_logs/2021-02-09T23_50_38_041Z-debug.log

> @remotion/cli@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/cli
> tsc -d && node add-bin.js

src/render.tsx(8,8): error TS2307: Cannot find module '@remotion/renderer' or its corresponding type declarations.
src/render.tsx(58,29): error TS7006: Parameter 'c' implicitly has an 'any' type.
src/render.tsx(76,19): error TS7006: Parameter 'f' implicitly has an 'any' type.

    at /Users/tagawahirotaka/Desktop/remotion/node_modules/execa/index.js:236:11
    at processTicksAndRejections (internal/process/task_queues.js:93:5) {
  code: 2,
  killed: false,
  stdout: '\n' +
    '> @remotion/cli@1.0.5 build /Users/tagawahirotaka/Desktop/remotion/packages/cli\n' +
    '> tsc -d && node add-bin.js\n' +
    '\n' +
    "src/render.tsx(8,8): error TS2307: Cannot find module '@remotion/renderer' or its corresponding type declarations.\n" +
    "src/render.tsx(58,29): error TS7006: Parameter 'c' implicitly has an 'any' type.\n" +
    "src/render.tsx(76,19): error TS7006: Parameter 'f' implicitly has an 'any' type.\n",
  stderr: 'npm ERR! code ELIFECYCLE\n' +
    'npm ERR! errno 2\n' +
    'npm ERR! @remotion/cli@1.0.5 build: `tsc -d && node add-bin.js`\n' +
    'npm ERR! Exit status 2\n' +
    'npm ERR! \n' +
    'npm ERR! Failed at the @remotion/cli@1.0.5 build script.\n' +
    'npm ERR! This is probably not a problem with npm. There is likely additional logging output above.\n' +
    '\n' +
    'npm ERR! A complete log of this run can be found in:\n' +
    'npm ERR!     /Users/tagawahirotaka/.npm/_logs/2021-02-09T23_50_38_041Z-debug.log\n',
  failed: true,
  signal: null,
  cmd: 'npm run build',
  timedOut: false,
  exitCode: 1
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @1.0.0-alpha.4 build: `lerna run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the @1.0.0-alpha.4 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tagawahirotaka/.npm/_logs/2021-02-09T23_50_38_095Z-debug.log
```

</details>

**Comment by wafuwafu13** [comment]
Should I have to add `// @ts-ignore`  in `packages/cli/src/render.tsx`?

<img width="935" alt="スクリーンショット 2021-02-10 9 08 42" src="https://user-images.githubusercontent.com/50798936/107445376-ae0c6a80-6b7f-11eb-8155-02c344b1dff2.png">

<img width="695" alt="スクリーンショット 2021-02-10 9 11 21" src="https://user-images.githubusercontent.com/50798936/107445614-27a45880-6b80-11eb-9be9-d66eacd6cbd6.png">

**Comment by JonnyBurger** [comment]
Sorry about this! I managed to reproduce and it's now fixed. The problem was a tsbuilinfo file being committed to the repository which messes with the Typescript cache.
With the newest main branch the problem doesn't occur.

Fixed by #32
